pynestml/utils/ast_nestml_printer.py

```diff
@@ -308,8 +308,8 @@ def print_declaration(self, node):
         ret += print_n_spaces(self.indent)
         if node.is_recordable:
             ret += 'recordable '
-        if node.is_function:
-            ret += 'function '
+        if node.is_inline_expression:
```

Comment:
This attribute should also be changed in the `ASTDeclaration` class. Some tests are failing due to this.

Reply:
Done!